voltdbclient/client.go

```diff
@@ -0,0 +1,240 @@
+/* This file is part of VoltDB.
+ * Copyright (C) 2008-2016 VoltDB Inc.
```

Comment:
If you leave a white line between the comment and any of the code in this case the `package` declaration this will not show in the docs.